charts/sonarqube-dce/templates/pod-disruption-budget.yaml

```diff
@@ -27,9 +32,15 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-{{- with .Values.ApplicationNodes.podDistributionBudget }}
-{{ toYaml . | indent 2 }}
-{{- end }}
+{{ if .Values.ApplicationNodes.podDistributionBudget }}
```

Comment:
Shouldn't be the other way round? If people define the `podDistruptionBudget`, this should have the precedence towards the deprecated value that is `podDistributionBudget`...

Reply:
i dont think so.

In terms of use cases new user will have the proper one coming from the default values.

But people that use the deprecated one, will also get an updated default value, and that default value should not be used for them so the first thing to check is if the deprecated one is being defined.

wdyt ?

Reply:
Yes, you're right. If we change the order here, the users having the old value will get it overridden, which is not what we want to do when deprecating...